ci/test_external.sh

```diff
@@ -81,6 +81,7 @@ set +e
 
 rapids-logger "running all gathered tests"
 DATASHADER_TEST_GPU=1 pytest --numprocesses=8 $FILES
+ --dist=worksteal \
```

Comment:
Adding linebreaks to make this align with other RAPIDS repos (easier to insert new flags programmatically in the future).
```suggestion
DATASHADER_TEST_GPU=1 pytest \
  --numprocesses=8 \
  --dist=worksteal \
  $FILES
```